clients/src/main/java/org/apache/kafka/clients/producer/internals/TransactionalRequestResult.java

```diff
@@ -52,11 +54,15 @@ public void await() {
     }
 
     public void await(long timeout, TimeUnit unit) {
```

Comment:
this method could be inlined to `await()`

Reply:
Also, `new KafkaException("Unknown reason.")` is a bit odd to me, Could you please refactor code to avoid such exception?